src/libvirtApi/common.js

```diff
@@ -378,20 +378,27 @@ function startEventMonitorStoragePools(connectionName) {
             switch (eventType) {
             case Enum.VIR_STORAGE_POOL_EVENT_DEFINED:
             case Enum.VIR_STORAGE_POOL_EVENT_CREATED:
+                logDebug(`StoragePoolEvent on ${connectionName} ${objPath}: DEFINED|CREATED`);
                 storagePoolGet({ connectionName, id: objPath });
                 break;
             case Enum.VIR_STORAGE_POOL_EVENT_STOPPED:
-                storagePoolUpdateOrDelete(connectionName, objPath);
+                logDebug(`StoragePoolEvent on ${connectionName} ${objPath}: STOPPED`);
+                storagePoolStopOrUndefine(connectionName, objPath);
                 break;
             case Enum.VIR_STORAGE_POOL_EVENT_STARTED:
+                logDebug(`StoragePoolEvent on ${connectionName} ${objPath}: STARTED`);
                 storagePoolGet({ connectionName, id: objPath, updateOnly: true });
                 break;
             case Enum.VIR_STORAGE_POOL_EVENT_UNDEFINED:
+                logDebug(`StoragePoolEvent on ${connectionName} ${objPath}: UNDEFINED`);
                 store.dispatch(undefineStoragePool({ connectionName, id: objPath }));
                 break;
             case Enum.VIR_STORAGE_POOL_EVENT_DELETED:
+                logDebug(`StoragePoolEvent on ${connectionName} ${objPath}: DELETED`);
+                // no need to handle
+                break;
             default:
-                logDebug(`handle StoragePoolEvent on ${connectionName}: ignoring event ${signal}`);
+                logDebug(`handle StoragePoolEvent on ${connectionName}: ignoring event ${eventType}`);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1285-20231102-144208-464cf7a9-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)